integration/spark-common/src/main/scala/org/apache/spark/util/FileUtils.scala

```diff
@@ -73,7 +73,8 @@ object FileUtils {
       val stringBuild = new StringBuilder()
       val filePaths = inputPath.split(",")
       for (i <- 0 until filePaths.size) {
-        val fileType = FileFactory.getFileType(filePaths(i))
+        val filePath = CarbonUtil.checkAndAppendHDFSUrl(filePaths(i))
```

Comment:
This is only for HDFS, right? How about other storage system support like S3?
@SangeetaGulia Can you have a look at this, I think this may impact #1584 that you are working on

Reply:
@jackylk I have verified this. It is working fine with S3 also. We will now be able to use the carbon property **carbon.ddl.base.hdfs.url** for s3 also to provide base URL.

Reply:
ok